--- a/pppd/ipcp.c
+++ b/pppd/ipcp.c
@@ -1778,6 +1778,7 @@
     ipcp_options *ho = &ipcp_hisoptions[f->unit];
     ipcp_options *go = &ipcp_gotoptions[f->unit];
     ipcp_options *wo = &ipcp_wantoptions[f->unit];
+    int ifindex;
 
     IPCPDEBUG(("ipcp: up"));
 
@@ -1893,9 +1894,19 @@
 	}
 #endif
 
+	ifindex = if_nametoindex(ifname);
+
 	/* run the pre-up script, if any, and wait for it to finish */
 	ipcp_script(_PATH_IPPREUP, 1);
 
+	/* check if preup script renamed the interface */
+	if (!if_indextoname(ifindex, ifname)) {
+	    if (debug)
+		warn("Interface index %d changed but can't find it");
+	    ipcp_close(f->unit, "Interface configuration failed");
+	    return;
+	}
+
 	/* bring the interface up for IP */
 	if (!sifup(f->unit)) {
 	    if (debug)
